### PR TITLE
[UT] mv compression ut to full test

### PR DESCRIPTION
**Pull request by J-shang**
#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [ ] full test - compression

### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: pipelines/full-test-compression.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ resources:
       endpoint: github-filter-connection
 
 variables:
-  filter.modified.globs: 'examples/model_compress/**,nni/algorithms/compression/**,nni/compression/**'
+  filter.modified.globs: 'examples/model_compress/**,nni/algorithms/compression/**,nni/compression/**,pipelines/full-test-compression.yml,test/algo/compression'
```
Comment (ultmaster) [inline pipelines/full-test-compression.yml:16]: ** needed?

**Review by ultmaster — APPROVED** [review]

File: pipelines/full-test-compression.yml (hunk 1 above)
Comment (J-shang) [reply pipelines/full-test-compression.yml]: nice catch, fixed

**Comment by liuzhe-lz** [comment]
The pipeline works well on master. Seems the error is caused by this PR.

**Comment by ultmaster** [comment]
The error is caused by missing `__init__.py`. I've already introduced one in #4933. Maybe you can wait for a while.

**Review by liuzhe-lz — APPROVED** [review]

File: pipelines/full-test-compression.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ resources:
       endpoint: github-filter-connection
 
 variables:
-  filter.modified.globs: 'examples/model_compress/**,nni/algorithms/compression/**,nni/compression/**'
+  filter.modified.globs: 'examples/model_compress/**,nni/algorithms/compression/**,nni/compression/**,pipelines/full-test-compression.yml,test/algo/compression/**'
```
Comment (liuzhe-lz) [inline pipelines/full-test-compression.yml:16]: This is really long. Can we use line break here?

**Pull request merged**